### DISCUSSION TITLE
Couldn't load Object [Only on android] & Y position Manipulation in AR

### DISCUSSION THREAD

**Discussion by YannickBoldly** [google/model-viewer#3268] [Q&A]
Hi all! 

We're facing some issues with the model viewer, but only on android; it won't open the AR model there. 
We've exported our model from the editor and uploaded it on our hosting, but we can't seem to get it to work. 

We're getting the "couldn't load object" message. 
You can see our implementation here;
https://boldlyxr-development2.nl/lamp
Additionally, I'd like to know if there is a way to manipulate height on android, on iOS you can manipulate the models height by swiping up with 2 fingers, but there doesn't seem to be such a feature on android.

**Comment by elalish** [comment]
Which `ar-mode` are you using? I believe you'll only get that error in scene-viewer, so have you tried webxr? Try narrowing your ar-modes to a single one so you can tell if it's simply failing for some reason (for instance WebXR requires HTTPS). As long as you don't set `ar-scale="fixed"` then you should be able to scale the model with two finger-pinch.

**Reply by YannickBoldly** [reply]
Hi @elalish we are using scene-viewer, because we think that one is fairly more user friendly then the barebones webXR for the unitiated user.  We think we have narrowed it down to a hosting issue, but we're serving the model via https, so I'm not sure what the issue is. 

What I meant with Y position manipulation was actually "lifting" the product up along the Y axis, on iPhone you can drag upwards with 2 fingers which will accomplish that. This is very beneficial for many types of AR, in this case light fixtures.
